integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/endpoints/EndpointsWithTrailingSlashTestCase.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.tests.testcases.standalone.endpoints;
+
+import com.github.dockerjava.zerodep.shaded.org.apache.hc.core5.http.HttpStatus;
+import io.netty.handler.codec.http.HttpHeaderNames;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.choreo.connect.tests.context.CCTestException;
+import org.wso2.choreo.connect.tests.util.ApictlUtils;
+import org.wso2.choreo.connect.tests.util.HttpResponse;
+import org.wso2.choreo.connect.tests.util.HttpsClientRequest;
+import org.wso2.choreo.connect.tests.util.Utils;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class EndpointsWithTrailingSlashTestCase {
```

Comment:
Can we move this test case to the APIM test suite

Reply:
In APIM 4.0.0 this issue is not reproducing. That's why I implemented this test case in the standalone test suite.

Reply:
Issue was reproduced in the latest APIM 4.0.0 pack. Fixed via https://github.com/wso2/product-microgateway/pull/2167/commits/9a417cff52b8ac407fb20039e5c40c8e5639489f